config/initializers/extends.rb

```diff
@@ -199,4 +199,18 @@ class Extends
     edit_wopi_file_on_result: 99,
     edit_wopi_file_on_step: 100
   }.freeze
+
+  ACTIVITY_GROUPS = {
+    projects: [*0..7],
+    task_results: [*23..26, 40, 41, 42, 99],
+    task: [*8..14, 35, 36, 37, 53, 54, *58..69],
+    task_protocol: [*15..22, 45, 46, 47, 38, 39, 100],
+    task_inventory: [55, 56],
+    experiment: [*27..31, 57],
+    reports: [48, 49, 50],
+    inventories: [*70..78],
+    protocol_repository: [*79..91],
+    team: [*92..96],
+    exports: [*97..98]
+  }.freeze
```

Comment:
Hm.. Let's check if we can get activity group from 'subject', because this is really hard to maintain